accumulator/forest.go

```diff
@@ -93,59 +93,57 @@ type Forest struct {
 	timeInVerify time.Duration
 }
 
+// forestType defines the 4 type of forests:
+// DiskForest, RamForest, CacheForest, CowForest
+type forestType int
```

Comment:
forestType should be capitalized since it needs to be accessible by callers that are importing package `accumulator`. So at the moment I don't think `cmd/utreexoserver` and `cmd/utreexoclient` builds (I've found other build errors but that's because of some dependency thing we have).

Reply:
For `cmd/utreexoserver` to build we will have to change the occurrences of `NewForest` function in utcd to accept the new arguments.

For now should I just change the `type forestType int` to `type ForestType int`?